docs/data/data-core-editor.md

```diff
@@ -1514,7 +1514,10 @@ inserted, optionally at a specific index respective a root block list.
  * block: Block object to insert.
  * index: Index at which block should be inserted.
  * rootClientId: Optional root client ID of block list on which
-                              to insert.
+                                   to insert.
```

Comment:
Maybe we should just not add a line break at all for arguments with long lines (to avoid messing with the markdown output)